Packs/Whois/Integrations/Whois/README.md

```diff
@@ -9,10 +9,14 @@ This integration was integrated and tested with version 1.0 of Whois
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
-    | Return Errors |  | False |
+    | Return Errors | If set, failed command results will be returned as warnings instead of errors. | False |
```

Comment:
The intention is that even if it encounters an error it will continue and not crash the whole program by returning the error as a warning
For multiple domains eg